### ISSUE TITLE
Monthly Ticket for Quest Changes

### ISSUE THREAD

**Issue by chochem** [GTNewHorizons/GT-New-Horizons-Modpack#11981]
### Your GTNH Discord Username

chochom#0271

### Your Pack Version

2.2.7/2.2.8

### Overview

A ticket for any large or small problems with any quests. Just post below. Of course, you can also make a dedicated ticket. In particular for more indepth suggestion or problems.

The most up-to-date quest files can always be found here: https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/blob/master/config/betterquesting/DefaultQuests.json

Below are the remaining things from last month (https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11765):

### Quest Changes

- [ ] **Quest.1305 For Fluid Recipes**

  It's better to mention that "AE2FC do fluid recipes better" or something in this quest, imo. EC2 is slow and buggy.

- [ ] https://github.com/GTNewHorizons/GT5-Unofficial/pull/1146 reworked turbine materials, so quests related to turbine materials need to be checked to confirm if any changes are needed.

- [x] **Quest.1904 Helium is Not Just for Balloons**
  > It's probably better to get it from end stone dust or monazite from the Moon.

  https://github.com/GTNewHorizons/GT5-Unofficial/pull/1387 improved the helium output from cracked refinery gas, so refinery gas is also a good source of helium now.

- [x] **Quest.3001 Using All the Fractions**

  Description about refinery gas needs updating.

- [x] **Quest.3014 Raw Gasoline**
  > An important detail here is that Raw Gasoline is 80% Naphtha, and none of the oil types found in reservoirs produces that much of it.

  Now raw oil can.

- [x] **Quest.3020 Largest Oil Drain**
  > until now. With a few stacks of Tungstensteel, you can put together the tier 3 of the Oil Rigs, and suck up all the fluid from a reservoir at once, covering the full 8x8 chunk size from a single spot.

  Tier 4. :(

- [x] **Quest.3028 Hyperheated Steam**
  > You will need four turbine items for each multi, and you best bring the best ones you can make!

  Not four, it's 12.

- [x] **Quest.3133 Nitrobenzene**
  > As a side note, Nitrobenzene shares an upgrade with Cetane-Boosted Diesel. They can both be used to make Super Fuels, which require more processing but are even more powerful.

  Why nitrobenzene can make super fuel?

- [ ] **Quest.3165 UV Power Per Turbine**
  > Large Ichorium 170% 342,720 EU/t (very durable)

  Ichorium turbine has been buffed. So this data is outdated.

- [ ] **Quest.3166 Powering-up Fluid Nukes**
  > Oriharukon and MAR-Ce-M200 are your best choices.

  MAR-Ce-M200 is LuV (need ZPM tier EBF) now. See https://github.com/GTNewHorizons/GoodGenerator/blob/33681ff5f240800c70b70c1e314d73ae262caf7c/src/main/java/goodgenerator/loader/RecipeLoader_02.java#L1145

- [x] **Quest.2996 Oil Reservoirs**
  description of raw oil needs update.

- [ ] **Quest.2568 Infinity Helmet** and **Quest.2569 Infinity Breastplate**
   Infinity Helmet can't remove all negative debuffs, it's the ability of Infinity Breastplate.

- [x] update Tungstensteel coil quest to TPV

- [x] Quest 2359 (Visit the Infested Islands) and Quest 2361 (Ghost Amulet)
The Infested Islands quest requires Ectoplasm, but that is only obtainable after getting the Ghost Amulet, and the Ghost Amulet quest is locked behind the Infested Islands one. Probably move the Ectoplasm item requirement to the Ghost Amulet quest.

- [x] Quest 2361 (Ghost Amulet)
The quest mentions needing to do a TC Infusion to purify the Amulet, but that is gated behind Draconium Ingots and the actual intended way of activating it is through a HEE Dragon Essence Altar infusion. Don't have to spell it out, maybe just add a hint about checking your HEE guide for alternative options.

- [x] Quest 425 (Elemental Chalk) Chalk does not require an orb to craft, and the Ritual Diviner is needed for Ritual Dismantler. Suggestion, move Quests 425 (Elemental Chalk) and 428 (Rituals Made Easy) between Quests 433 (Rituals) and 2462 (Ritual Dismantler).

- [x] Quest 927 (Blood Altar Tier 4) Since Large Bloodstones need the tier 3 orb, Quest 424 (Magican's Orb) should really be between it and Quest 421 (Tier 3 Slates).
These changes would probably require the Blood Magic tab to be reorganised slightly though.

- [x] Quest 1102 (Beacon) The beacon is a fusion crafting recipe, and can be crafted as soon as you have access to fusion crafting and have killed the wither, it does not need a Silverwood Wand. Suggest moving it left of Quest 265 (Creating a Better Wand).

- [x] Quest 2268 (Who Needs Friends?) Luggage only needs 75 vis to craft so can be crafted with any wand that requires Energetic Screws, suggest moving the requirement of 2268 to be quest 266.

- [x] Quest 2068 (Disenchanted) Same as above.

- [x] Quests 303 (Pushing the Limits) and 304 (Enchanting Like a Pro) Are fusing crafting recipes and don't require a wand, but to keep things together should probably be moved with the above quest.

- [x] The chlorine quest (and quests leading up to it) need to be made into main quests. (MV main quests are dependent upon learning these)

- [x] In the Flower Power tab, all quests involving flowers are can be completed by holding any botania functional or generating flower as they all have the same ID. I think ignoreNBT and partalMatch needs to be false for all these quests, quick test in creative seems to fix the problem.

- [x] Quest 1693 (2-Nitrochlorobenzene)
The Optional Task calls for 18 Nitration Mixture and 18 Chlorobenzene Cells, but the recipe using those is a 2:1 ratio. Up the amount of former to 36. (checked, this was already fixed)

### New Quests/Full Overhauls

- [ ] Add quests for AE2FC fluid cells in AE tab. They are good alternatives to EC2 ones.

- [x] Add a quest for quad core and 16 core co-processing unit in AE tab.

- [ ] Add a quest for Quadruple Input Hatch, or mention it in some multiblock quests (for example, EBF).

- [ ] Add a quest in Multblock Goals tab listing what GT tools can do with multiblock controllers. (Or add them to each multi's tooltip or quest) Examples are as follows:
  - Screwdriver: (Right Click) Toggle mode for some multis, lock recipe for LCR, set prioritise circuit for MEBF, toggle input bus seperation for PA, EBF, Industrial Wiremill etc. (Shift Right Click) Lock recipe for PA
  - Soldering Iron: (Right Click) Toggle void excess for GT++ multis
  - Wire Cutter: (Right Click) Toggle UEV+ treated as UHV for PA, toggle input bus seperation for AdvEBF, MEBF, ABS, etc.
  - Plunger: (Right Click) Clear output hatch for GT++ multis

- [ ] Add a quest for Integrated Ore Factory in Multiblock Goals tab (maybe add a link in UHV tab as well). It is the ultimate solution for ore processing in late game.

- [ ] Add a quest for Tesla Tower in Multiblock Goals tab. It is a powerful power transmission option in HV~ZPM that should be mentioned in questbook.

- [ ] Add a quest/quests for YOTtank in Multiblock Goals tab. It is the ultimate fluid storage solution in middle & late game.

- [ ] Add a quest for Infinite Oil/Gas/Fluid Drilling Rig. There is already a quest for void miner, so add one for Infinite Rig is reasonable.

- [ ] Add a quest for GT++ Solar Tower in How to Generate Power tab, it is now a power generation option in IV, since https://github.com/GTNewHorizons/GTplusplus/pull/290 fixed and reworked it.

- [ ] Add XL Gas Turbine and XL Plasma Turbine quests in How to Generate Power tab, they are good upgrade for the large ones. XL Steam/HP Steam Turbine have a quest in Working with Oil tab already, might need a link in How to Generate Power tab too?

- [ ] Add a quest for Turbine Housing in How to Generate Power tab, it makes automating large turbine possible.

- [ ] Add a quest for Elemental Duplicator in GT++ tab.

- [ ] Add quests for the Algae Farm. -- Regarding this point, the Algae Farm doesn't just need a quest, it needs an entire questline where it's properly balanced. It can replace several early automations for specific chemicals if used, so it is my (steelux) plan to make that questline when I finish the remaining adjustments.

- [x] Add a link of Steam Grinder quest (GT++ tab) in Tier 0.5 - Steam Age tab.

- [ ] Quest.1510 Biomes Everywhere, But Not the One You Want? Overhaul based on https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/11385#issuecomment-1257110174

- [x] add quests for graphene in EV

- [x] add quests for advanced glue

- [ ] add quests for super glue

- [x] add quest for the PCB factory

- [ ] add quest for the component assembly

- [ ] add quest for the QFT

- [x] add quest for the bee multiblock (IAADDS)

- [x] add quest for the space elevator

- [x] add quest for the energy infuser

- [ ] the Lutetium quests need an overhaul. Specifically, 'Faster Lutetium Excitement', 'High Temperature Super Breeder'. The neutron activator option was removed in favor of the HTGR.

**Comment by chochem** [comment]
- [ ] rewrite the TFFT quest(s) because of https://github.com/GTNewHorizons/KekzTech/pull/45

**Comment by Steelux8** [comment]
> reworked turbine materials, so quests related to turbine materials need to be checked to confirm if any changes are needed.

I found that not all the materials I changed are usable for turbines right now, so I'll have to fix all the related issues before I can document what the new good materials are.

> Why nitrobenzene can make super fuel?

This is what happens when you write quest descriptions before making the actual changes. I forgot to actually add Nitrobenzene as an option for Super Fuel...

> The quest mentions needing to do a TC Infusion to purify the Amulet, but that is gated behind Draconium Ingots and the actual intended way of activating it is through a HEE Dragon Essence Altar infusion. Don't have to spell it out, maybe just add a hint about checking your HEE guide for alternative options.

The questbook should not be ambiguous about information, the purpose of those quest description is to explain to players how things work.

> Add a link of Steam Grinder quest (GT++ tab) in Tier 0.5 - Steam Age tab.

I believe there is also a Steam Compressor multi at the same progression stage. That also needs a quest.

> add quests for advanced glue + graphene in EV

Advanced Glue comes before EV, and it might have other uses earlier on (I can't exactly remember). I have to check all the related recipes to find out where the quest should go.

> add quest for the QFT

This is also going to be a questline, not a single quest. The QFT is too complex to cover in one or two quests.

> the Lutetium quests need an overhaul. Specifically, 'Faster Lutetium Excitement', 'High Temperature Super Breeder'. The neutron activator option was removed in favor of the HTGR.

This is a change I'm dreading, because I don't know how the HTGR is actually viable for building at its stage in progression. Removing the thorium excited fuel step for Lutetium was not a good idea, and I don't know what to do about it.

**Comment by Rdjrjqouqcu** [comment]
Minor issue I just found, blood magic rune capacity was tweaked, it's `+3500` for augmented capacity, `*1.14` for superior capacity instead of what the quests say (`+2000` and `*1.1`).

**Comment by Hiagg** [comment]
"starting fusion" quest in luv tab asks for 32 luv casing when minimum is 79

**Comment by chochem** [comment]
@Rdjrjqouqcu this has been fixed for a while, you must be out of date

**Comment by ashido96** [comment]
"Smelt All the Things Stack-Wise..." 

The "Multi Smelter" multiblock requires HV circuits but the quest unlocks much earlier than that.

**Comment by Excavon** [comment]
Coke oven bricks quest recommends making all bricks before starting to smelt so that the player gets the quest, but the first task for unfired bricks is optional so they don't have to.

**Comment by Gofer2x** [comment]
Remaining Tungstensteel Coil mentions. TPV FTW.

- [ ] Quest 213 (Tier 5 (IV))
- [ ] Quest 1408 (EV Superconductors (2048 EU/t))
- [ ] Quest 3012 (Improved Oil Cracker)
- [ ] Quest 1522 (Niobium-Titanium)
- [ ] Quest 1718 (Yttrium-Barium-Cuprate)

**Comment by Gofer2x** [comment]
Most quests haven't been updated for the new SMD Inductors. I'll point out which ones have recipes that accept either the non-SMD or SMD version or just the latter, and should be changed to accept them accordingly.

- [ ] Quest 1017 (Processor Assembly) Needs either, but can you even get the Neodymium for the SMD version at that point? If not just keep it as it is. The quest is also missing the 8 Capacitors in its Optional Task (intentional?).
- [ ] Quest 1225 (Mainframes, Your First IV Circuit) Needs either.
- [ ] Quest 1250 (Nanoprocessor Array) Needs SMD, also change choice reward. Also also the quest name and description are wrong, the item is called "Nanoprocessor Assembly".
- [ ] Quest 1261 (Nanoprocessor Mainframe) Needs SMD.
- [ ] Quest 1705 (Quantumprocessor Assembly) Needs SMD, also change choice reward.
- [ ] Quest 1707 (Quantumprocessor Mainframe) Needs SMD, also change choice reward.
- [ ] Quest 1708 (Crystalprocessor Assembly) Needs Advanced SMD.

- [ ] SMD Inductors become mandatory at around the Nanoprocessor Array/Assembly, so probably give them a quest somewhere around there.

- [ ] Quest 2608 (Advanced SMDs) Add Advanced SMD Inductors to the Retrieval Task.

**Comment by snoudin** [comment]
Is everything fine with sunnarium quest line in EV?
It seems very strange for me.
IC molecular transformer is disabled and uncraftable if to trust NEI (but in quest 'Quit Exhaling Dusts' noted as craftable in QED). And there IS quest to make it.
GT molecular transformer can't be crafted without assemblyline (so, at leats before end of IV). 
But there is quest for sunnarium battery?
What do I miss?

**Comment by chochem** [comment]
> What do I miss?

I think the only thing you are missing is that you are on a fairly outdated version. sunnarium is EV from nukes, the ++ multi is LuV

**Comment by snoudin** [comment]
> > What do I miss?
> 
> I think the only thing you are missing is that you are on a fairly outdated version. 

I am on 2.2.8 from here currently:
http://downloads.gtnewhorizons.com/Multi_mc_downloads/

**Comment by chochem** [comment]
> > > What do I miss?
> > 
> > 
> > I think the only thing you are missing is that you are on a fairly outdated version.
> 
> I am on 2.2.8 from here currently: http://downloads.gtnewhorizons.com/Multi_mc_downloads/

your questbook has not been updated in like 8 months though.

\bq_admin default load

**Comment by snoudin** [comment]
Thanks mate. Appreciate your help

**Comment by minecraft7771** [comment]
Parametrizer quest should be updated to state that the Parametrizer is no longer needed. Parameters can now be edited in the GUI by blicking on the corresponding LED. Existing Parametrizers still work and will overwrite the values.

**Comment by minecraft7771** [comment]
Quest about the SpaceSuit could be edited to reflect the new behavior of many different armors giving the SpaceSuit effect.

**Comment by minecraft7771** [comment]
https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/12184

**Comment by ashido96** [comment]
I have no clue what it does. But crafting recipe for "Network Latency Pipe" is disabled so it probably should not be offered as a quest reward. I assume it has unfixed bugs (on 2.2 anyway). 
![image](https://user-images.githubusercontent.com/2313143/210402179-b03bc9bd-0d98-4977-92f7-397cab62fc0b.png)

**Comment by Glease** [comment]
- [ ] Correct/clarify catalyst damage chance of each pipe casing in chemplant
Bronze pipe 100%
Steel pipe 80%
Titanium 60%
Tungstensteel 40%

Preferably also clarify that cupronickel coil is really double the normal recipe duration, instead of +50% speed.

**Comment by Hiagg** [comment]
quest "Space Station", #1067, in "Space Race" tab says "Naptune - T6" instead of "Neptune - T6". this is extremely immersion breaking and i am having a hard time playing the pack

**Comment by chochem** [comment]
@ashido96 havent heard of any bugs there

**Comment by ashido96** [comment]
> 

You literally can't craft the item. It is only accessible through this quest.

**Comment by chochem** [comment]
@ashido96 I am not gonna remove it just because you speculate that there could be a problem, maybe. While neither of us even knows what it does.

**Comment by ashido96** [comment]
![image](https://user-images.githubusercontent.com/2313143/211616905-d2a7366a-f46c-4a6a-8284-5c68a0d5ba15.png)
Can you please write the "(after a couple steps)" for this quest please.

**Comment by chochem** [comment]
@ashido96 you have multiple options there, so its really up to you. you can start by making sulfur, sulfur dioxide, or diluted sulfuric acid. Either option can get you there.

**Comment by shadowpho** [comment]
HSS-S coils are "required" to progress into LuV.

However I have naquadah coils, and the questbook for HSS-S coils says they are optional. 
![image](https://user-images.githubusercontent.com/217332/211940333-9c4e2a0a-c492-4dc1-b434-0c986a826227.png)

**Comment by SPMoo66** [comment]
- [ ] Quest.1670 Naquadah Doped Monocrystalline Silicon
Gallium Arsenide Crystal requirement should be increased to 4 to create the 4 boules for the task

**Comment by minecraft7771** [comment]
Sorry I couldn't comment on the PR before it was merged about the space elevator quest.
1. The info about the rocket control computer will be outdated in the next dev and stable release. The rocket control computer was replaced by another parameter "Planet type"
2. Maybe its worth mentioning that the elevator also allows planetary travel (But you will need to bring your own means of teleportation or rocket to get back)
3. The mining module has a cycle mode added. Idk if thats too much in depth or not but you can basically switch between static and cycle mode. In static it will hold its distance while in cycle it will change its distance after each recipe check (and you can configure the rang and step size of that)

**Comment by chochem** [comment]
feel free to write something yourself. I just grabbed the info from the wiki, so that there is a quest.

**Comment by chochem** [comment]
By which I mean that I have no idea how the teleportation or this cycle mode works, so I cant do it.

**Comment by Hiagg** [comment]
Quest 2395 (Formaldehyde Catalyst) has a slight grammar and spelling issue: "mixing iron and vanadium, and stuffing into carrior"

**Comment by minecraft7771** [comment]
More of suggestion:
Quest 2648 (The Final Frontier) is required in UHV to progress further. For skyblock and the mad people who want to skip T8 this should either accept T8 or the Space Elevator controller (maybe again with the hint that you need teleporters to get back).

**Comment by chochem** [comment]
> More of suggestion: Quest 2648 (The Final Frontier) is required in UHV to progress further. For skyblock and the mad people who want to skip T8 this should either accept T8 or the Space Elevator controller (maybe again with the hint that you need teleporters to get back).

thats a checkbox task lol. I think people can manage that.

**Comment by chochem** [comment]
ah, you mean the requirements. I guess you only heard someone complaining broadly.
- [ ] change final frontier reqs

**Comment by minecraft7771** [comment]
The Quest Awakened Draconium Coil Upgrade #1833 also needs addressing. It currently has Rocket Tier 8 as requirement. Imo it can be changed to have Final Frontier as requirement.

**Comment by minecraft7771** [comment]
Lasers Solve Everything 2730. It says the loss is 1 EU/t per max ampere. However its 1 EU/s

**Comment by CatsLeftEar** [comment]
Rhodim #2328
doesnt accept both chlorine and ammonia cell, retrieval task

**Comment by chochem** [comment]
unlikely, but will test later. you sure you have the prereqs are not in edit mode?

**Comment by Ethryan** [comment]
Maybe move the TConstruct Reinforcement modifier from steam age to MV after fluid solidifier to adress the change in requirement for the modifier.

**Comment by chochem** [comment]
just tested, Rhodium works fine, just as expected.

**Comment by AbdielKavash** [comment]
![image](https://user-images.githubusercontent.com/19243993/216631625-44786d2b-d418-44fb-8f73-bfe7e323d7bb.png)

This is incorrect; you get one unique bag as a reward for each piece of Kami armor, and another for the LEG. You can get 5 unique bags even without trading once.
